coderd/database/pubsub/pubsub.go

```diff
@@ -188,6 +187,19 @@ func (l pqListenerShim) NotifyChan() <-chan *pq.Notification {
 	return l.Notify
 }
 
+type queueSet struct {
+	m map[*msgQueue]struct{}
```

Comment:
Suggestion: m => q/queues. I feel this would explain better what one is looking at when viewing the code.